package.json

```diff
@@ -39,9 +40,8 @@
     "release:beta-from-package": "yarn release from-package --conventional-prerelease --preid beta --pre-dist-tag beta --no-private",
     "release:site": "gulp prepareSite && gh-pages -d dist-site/ -f -e .",
     "start": "NODE_ENV=development yarn workspace @spectrum-css/preview start",
-    "docs:mod": "node tasks/mod-extractor.js",
-    "test": "VRT=true run-p dev \"test:run {1}\" --",
-    "test:run": "yarn workspace @spectrum-css/test run test",
+    "test": "yarn workspace @spectrum-css/preview test",
```

Comment:
I added a `scope` command to let us run chromatic for a single or subset of components - should help us save runs. I also removed the `:visual` identifier since (for now), we don't have any other testing tools. Just makes it more succinct since visual is kind of a weird word to spell.